packages/mui-material-next/src/Select/Select.d.ts

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { SxProps } from '@mui/system';
+// TODO v6: replace @mui/material with @mui/material-next when components are available
+import { InternalStandardProps as StandardProps, Theme } from '@mui/material';
+import { InputProps } from '@mui/material/Input';
```

Comment:
🤔 Not necessarily, though looking at the usage in this file:

```ts
export interface SelectProps<Value = unknown>
  extends StandardProps<InputProps, 'value' | 'onChange'>,
    Omit<OutlinedInputProps, 'value' | 'onChange'> {
   // etc
}
```

It seems to be roughly the same as:

```ts
export interface SelectProps<Value = unknown>
  extends StandardProps<InputBaseProps, 'value' | 'onChange'> {
    sx?: SxProps<Theme>; // common
    // all the props supported at the at the variant component level
    hiddenLabel?: boolean; // for FilledInput
    disableUnderline?: boolean; // for FilledInput + StandardInput
    label?: React.ReactNode; // for OutlinedInput
    notched?: boolean; // for OutlinedInput
    // the select props
}
```

So it could maybe be rewritten using `InputBaseProps`, `FilledInputProps` and `OutlinedInputProps`

Reply:
That makes sense. I'll add this to the Select issue and make the TODO comments more specific.